test/install/CMakeLists.txt

```diff
@@ -8,8 +8,16 @@ if(CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME)
     if(NOT BUILD_SHARED_LIBS)
         set(SFML_STATIC_LIBRARIES TRUE)
     endif()
+
+    # cmake by default will only search inside the iOS SDK for packages/libraries, so we need to tell it to look elsewhere
+    if (IOS)
+        set(CMAKE_FIND_ROOT_PATH_MODE_PACKAGE BOTH)
+        set(CMAKE_FIND_ROOT_PATH_MODE_LIBRARY BOTH)
+    endif()
+
     find_package(SFML 3.0.0 EXACT CONFIG REQUIRED COMPONENTS Graphics Network Audio)
 endif()
 
 add_executable(test-sfml-install Install.cpp)
 target_link_libraries(test-sfml-install PRIVATE SFML::Graphics SFML::Network SFML::Audio)
+message(STATUS ${CMAKE_OSX_SYSROOT})
```

Comment:
Why this message?

Reply:
It was mostly useful for diagnosing SDK compatibility issues so you can see which one is being used - happy to take any direction on whether it should be removed or formatted differently